### DISCUSSION TITLE
Is it allowed to ask for new packages?

### DISCUSSION THREAD

**Discussion by ghost** [msys2/MSYS2-packages#4362] [General]
I don't see the `Package request` issue template like on https://github.com/msys2/MINGW-packages.

**Comment by Biswa96** [comment]
> I don't see the `Package request` issue template like on https://github.com/msys2/MINGW-packages.

Despite some of these central parts being based on Cygwin, the main focus of MSYS2 is to provide a build environment for native Windows software and the Cygwin-using parts are kept at a minimum.[^1]

[^1]: https://www.msys2.org/
